ADAL/src/cache/ADTokenCacheAccessor.m

```diff
@@ -87,6 +87,9 @@ - (ADTokenCacheItem *)getItemForUser:(NSString *)userId
                                                       userId:userId
                                                correlationId:[context correlationId]
                                                        error:&adError];
+        item.storageAuthority = item.authority;
+        item.authority = _authority;
+        
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3AS1131)